components/tools/OmeroPy/test/integration/test_tickets3000.py

```diff
@@ -68,11 +68,6 @@ def test2396(self):
         l_ia.setChild(fa)
         self.update.saveObject(l_ia)
 
-    # This test is no longer valid as it should not be possible to remove
```

Comment:
It was originally marked as `xfail` because it failed. Looking at the logic at the time I couldn't see how it could pass. Now looking at the logic it seems reasonable and now it passes. Maybe it was a test waiting for a server change!